modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsImpl.java

```diff
@@ -311,6 +325,13 @@ public CacheMetricsImpl(GridCacheContext<?, ?> cctx, boolean isNear) {
 
         rebalanceClearingPartitions = mreg.longMetric("RebalanceClearingPartitionsLeft",
             "Number of partitions need to be cleared before actual rebalance start.");
+
+        getHistogram = mreg.histogram("GetHistogram", HISTOGRAM_BUCKETS, "Get duration histogram in nanoseconds.");
```

Comment:
Can we track new histograms in milliseconds?
I think we should have similar units for all metrics(other time metrics tracked in milliseconds).

Reply:
I think we should not measure it in millisecond because most of all operations are completed for hundreds of microseconds.

Reply:
OK. Let's keep it in nanoseconds.